INSTALL.make.md

```diff
@@ -0,0 +1,257 @@
+   and coqc selects a particular binary.
+
+    * `coqtop' launches Coq in the interactive mode. By default it loads
+      basic logical definitions and tactics from the Init directory.
+
+    * `coqc' allows compilation of Coq files directly from the command line.
+      To compile a file foo.v, do:
+
+        coqc foo.v
+
+      It will produce a file foo.vo, that you can now load through the Coq
+      command "Require".
+
+   A detailed description of these commands and of their options is given
+   in the Reference Manual (which you can get in the doc/
+   directory, or read online on http://coq.inria.fr/doc/)
+   and in the corresponding manual pages.
+
+
+COMPILING FOR DIFFERENT ARCHITECTURES.
+======================================
+
+   This section explains how to compile Coq for several architecture, sharing
+   the same sources. The important fact is that some files are architecture
+   dependent (.cmx, .o and executable files for instance) but others are not
+   (.cmo and .vo). Consequently, you can :
+
+   o  save some time during compilation by not cleaning the architecture
+      independent files;
+
+   o  save some space during installation by sharing the Coq standard
+      library (which is fully architecture independent).
+
+   So, in order to compile Coq for a new architecture, proceed as follows:
+
+   * Omit step 7 above and clean only the architecture dependent files:
+     it is done automatically with the command
+
+        make archclean
+
+   * Configure the system for the new architecture:
+
+        ./configure <options>
+
+     You can specify the same directory for the standard library but you
+     MUST specify a different directory for the binaries (of course).
+
+   * Compile and install the system as described in steps 5 and 6 above.
+
+
+MOVING BINARIES OR LIBRARY.
+===========================
+
+   If you move both the binaries and the library in a consistent way,
+   Coq should be able to still run. Otherwise, Coq may be "lost",
+   running "coqtop" would then return an error message of the kind:
+
+       Error during initialization :
+       Error: cannot guess a path for Coq libraries; please use -coqlib option
+
+   You can then indicate the new places to Coq, using the options -coqlib :
+
+       coqtop -coqlib <new directory>
+
+   See also next section.
+
+
+DYNAMICALLY LOADED LIBRARIES FOR BYTECODE EXECUTABLES.
+======================================================
+
+   Some bytecode executables of Coq use the OCaml runtime, which dynamically
+   loads a shared library (.so or .dll). When it is not installed properly, you
+   can get an error message of this kind:
+
+    Fatal error: cannot load shared library dllcoqrun
+    Reason: dllcoqrun.so: cannot open shared object file: No such file or directory
+
+   In this case, you need either:
+     - to set the CAML_LD_LIBRARY_PATH environment variable to point to the
```

Comment:
I guess you need to skip lines for this to render properly. But also, please remove all the indentation of normal text within a section.

Reply:
Done, thanks for the feedback.